### ISSUE TITLE
Symfony 2.7

### ISSUE THREAD

**Issue by progmancod** [codeconsortium/CCDNUserSecurityBundle#51]
The composer.json imposes a limit of "symfony/symfony": "~2.5" - is the upper bound strictly necessary? 

Symfony 2.7 is the current LTS.

**Comment by reecefowell** [comment]
not really, as long as it works. Have you tested it? As long as it works with the most of the later versions.

If so, send a PR. I just merged one to fix what I thought was this problem. I don't really do any work on this bundle anymore. But if you want to submit a PR I will merge it.

**Comment by kikegarcia** [comment]
I created a fork for testing it on 2.7 and it's running fine. I'll check 2.8 (new LTS) soon too.
